container/go/cmd/puller/puller.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//////////////////////////////////////////////////////////////////////
+// This binary pulls images from a Docker Registry.
+// Unlike regular docker pull, the format this package uses is proprietary.
+
+package main
+
+import (
+	"flag"
+	"log"
+	ospkg "os"
```

Comment:
I would not rename the standard package as it hinders readability. I would rename the variable that captures the "os" flag something else like "osVar". imo, readers of a language get quite used to the standard library and their API and having one renamed hinders readability more than a command line variable renamed to avoid a name conflict.

However, this is just my opinion and you may choose to disagree :)

Reply:
Just spoke with @xingao267. Ignore this comment